scraper.rb

```diff
@@ -2,239 +2,38 @@
 
 require 'dotenv'
 Dotenv.load
-require 'octokit'
-require 'scraperwiki'
-require 'mechanize'
-require 'rest-client'
-require 'json'
-require 'active_support/all'
-
-# Duck punches
-class Numeric
-  def percent_of(n)
-    to_f / n.to_f * 100.0
-  end
-end
-
-# PlanningAlerts contributor stats from GitHub
-class GitHub
-  class << self
-    def commits_text(period:)
-      puts 'Collect information from GitHub'
-      if commits_count(period: period).zero?
-        nil
-      else
-        "You shipped #{commits_count(period: period)} commits in the same period."
+require_relative 'lib/github'
+require_relative 'lib/planningalerts'
+require_relative 'lib/righttoknow'
+require_relative 'lib/runner'
+
+module Jacaranda
+  # The runner for PlanningAlerts
+  class PlanningAlerts < Runner
```

Comment:
This is great feedback, thanks @equivalentideas. 

I've put up another PR at #8 to experiment with the approach you've described above. 

It definitely makes the code cleaner and easier to read about. Also, the tests suddenly make a lot more sense. 

In that PR I've opted to retain the existing runner loading, but have tried to make it a bit clearer what's happening by adding a `--list-runners` option.